mailpoet/lib/Cron/Workers/SendingQueue/SendingQueue.php

```diff
@@ -273,7 +280,12 @@ public function processQueue($queue, $newsletter, $subscribers, $timer) {
       $preparedSubscribersIds[] = $subscriber->id;
       // create personalized instant unsubsribe link
       $unsubscribeUrls[] = $this->links->getUnsubscribeUrl($queue, $subscriber->id);
-      $metas[] = $this->mailerMetaInfo->getNewsletterMetaInfo($newsletter, $subscriber);
+
+      $subscriberEntity = $this->subscribersRepository->findOneById($subscriber->id);
+      if ($subscriberEntity instanceof SubscriberEntity) {
```

Comment:
Can we fill `$metas[]` with `[]` if this condition returns `false`? Otherwise `$metas` will be out of sync with other arrays. I assume that same indexes should hold data for the same recipient.

Reply:
When working on this I assumed it was not necessary. This code shouldn't last long as it is only there while we are transitioning from Paris to Doctrine. Soon we should refactor \MailPoet\Cron\Workers\SendingQueue\SendingQueue::processQueue() and it will receive an array of subscriber entities and it won't be necessary to call findOneById() here anymore. I think that if we have a subscriber model and we can't find the equivalent subscriber entity we have much bigger problems.

But anyway, adding the else doesn't hurt and it won't create any problems. So I added it in a new commit. Could you please check this PR again?